completions/bash/cou_bash_completion

```diff
@@ -0,0 +1,61 @@
+_cou_bash_completion() {
```

Comment:
Can this be done also for zsh? It works fine for bash.

Reply:
Thanks for the suggestion. As discussed elsewhere, this will be handled later as a future enhancement. I opened an issue to track: #58